src/Form/UserType.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
You should consider deleting this empty class as it has no attributes or methods to be inherited to be further used.